client-v2/src/types/FaciaApi.ts

```diff
@@ -94,6 +97,23 @@ interface CollectionResponse {
   };
 }
 
+interface EditionCollectionFromResponse {
```

Comment:
I've put this here as I'm not sure it should be in `shared` but happy to reconsider.

Reply:
👍